server/lib/enrich/article.js

```diff
@@ -39,7 +39,10 @@ module.exports = exports = function article(content, contract, graphicSyndicatio
 			)
 
 	content.canAllGraphicsBeSyndicated =
-		graphicEmbeds &&
+		// From MDN docs: every acts like the "for all" quantifier in mathematics.
+		// In particular, for an empty array, it returns true.
+		// So if the filter above returned an empty array, [].every(whatever) would return `true`
+		graphicEmbeds.length > 0 &&
```

Comment:
When `graphicEmbeds` is undefined this will break. We need to keep the previous check